x-pack/plugins/ml/public/application/settings/anomaly_detection_settings.tsx

```diff
@@ -160,7 +165,79 @@ export const AnomalyDetectionSettings: FC = () => {
             </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlexItem>
-        <EuiFlexItem grow={5}>
+
+        <EuiFlexItem>
+          <EuiTitle size="s">
+            <h3>
+              <FormattedMessage
+                id="xpack.ml.settings.anomalyDetection.calendarsDstTitle"
+                defaultMessage="Calendars DST"
+              />
+            </h3>
+          </EuiTitle>
+          <EuiSpacer size="s" />
+          <EuiText size="s">
+            <EuiTextColor color="subdued">
+              <p>
+                <FormattedMessage
+                  id="xpack.ml.settings.anomalyDetection.calendarsDstText"
+                  defaultMessage="Calendars contain a list of scheduled events for which you do not want to generate anomalies, such as planned system outages or public holidays."
```

Comment:
@szabosteve could you please help with the text for this page.
Also the link here goes to our documentation which will need updating.
https://www.elastic.co/guide/en/machine-learning/master/ml-ad-run-jobs.html#ml-ad-calendars